### ISSUE TITLE
Testing accuracy on the fly during training

### ISSUE THREAD

**Issue by lyttonhao** [yjxiong/temporal-segment-networks#72]
Hi @yjxiong 

Since I am re-implementing TSN, could you provide the testing accuracy on the fly during training or your training log? So it is much more easier to compare training performance with your original model without evaluating it at last. Thanks a lot!

**Comment by yjxiong** [comment]
I suggest you instead compare the final evaluation results to check whether your implementation is correct. The on-the-fly validation only serves for monitoring and they are not run on the exact entire validation set each time. 

We have tried implementing TSN with several libraries and the final performance is stable.

**Comment by lyttonhao** [comment]
@yjxiong Thanks for your reply and your suggestion.  I understand your meaning. However, since I do not achieve your final results, I want to check which part (the training part or the final 10-crop evaluation) may have some bugs in my implementation.

Could you provide a rough number of the accuracy of the on-the-fly validation (1-crop of 3 segments)? Thus I could know about how much can the final 10-crop of 25 segments predict improve over the validation results during training. Thanks a lot!

**Comment by yjxiong** [comment]
For RGB with segment number 3 using the bundled Caffe, the on-the-fly number can be between 83% to 85%. For flow, it can also be something between 83-85. 
And as I said, this number is highly versatile, which means in different frameworks we observe different ones due to the validation settings. However, the video-level testing performance is quite stable across frameworks.

**Comment by lyttonhao** [comment]
@yjxiong Thanks a lot for your reply. It helps me a lot. I have another question about the pre-trained model. In order to be the same as your setting, I want to use the same original Inception-BN model which was pre-trained on ImageNet. Did you use the public Inception-BN re-implemented model? If so, could you provide the link? Many thanks for your help!

**Comment by yjxiong** [comment]
As we know, there is no public implementation of BN-inception that matches the originally reported performance. The one we released as the RGB initialization model is our own implementation which achieves the reported accuracy. You can use it as you will.

**Comment by lyttonhao** [comment]
Thanks for your reply. @yjxiong
